app/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalSecrets.java

```diff
@@ -0,0 +1,43 @@
+package gov.hhs.cdc.trustedintermediary.external.localfile;
+/**
+ * This Class implements the Secret interface, and it's purpose is to retrieve secrets from our
+ * local environment
+ */
```

Comment:
Class level JavaDocs should show up right above the `class`, not above the imports.

Reply:
Good catch!